constants.py

```diff
@@ -6,4 +6,10 @@
 TELEGRAM_API_TOKEN = env('TELEGRAM_API_TOKEN')
 GITHUB_SECRET = env('GITHUB_SECRET')
 
-CONVERSATIONS = env.list('CONVERSATIONS', [131845033, -1001342903759], subcast=int)
\ No newline at end of file
+CONVERSATIONS = env.list('CONVERSATIONS', [519013710], subcast=int)
```

Comment:
Please revert this to the original line, so that we'll alert the correct chat IDs.